app.py

```diff
@@ -0,0 +1,18 @@
+import os
+from flask import Flask, render_template
+
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def index():
+    return render_template('pages/index.html')
+
+
+if __name__ == "__main__":
+    app.run(
+        host = os.environ.get('IP', '127.0.0.1'),
+        port = os.environ.get('PORT', '5000'),
```

Comment:
Should this line be an int? : port=int(os.environ.get("PORT")),
Or is that only if we don't specify the actual PORT number within the parentheses?

Reply:
@StephenJ2020 I don't think we need to specify data types when setting environment variables.